pipes/WDL/workflows/demux_deplete.wdl

```diff
@@ -164,6 +164,7 @@ workflow demux_deplete {
         File        spikein_counts                           = spike_summary.count_summary
         
         String      run_date                                 = illumina_demux.run_info[0]['run_start_date']
+        String      instrument_model                         = select_first(flatten([illumina_demux.instrument_model,[""]]))
```

Comment:
`miniwdl check` is erroring in travis because it doesn't like workflow output variables having the same name as workflow input variables -- perhaps change the name slightly?

Reply:
This was residual, but I'll keep it as an override—much better. Thanks for taking a look at this.

Reply:
I think the continuing CI issue though is that the output variable name is the same as the input variable name. Maybe change the outputs to `instrument_model_inferred` or something?